db/src/bootstrap.rs

```diff
@@ -233,27 +257,27 @@ fn symbolic_schema_to_assertions(symbolic_schema: &Value) -> Result<Vec<Value>>
 }
 
 pub fn bootstrap_partition_map() -> PartitionMap {
-    V2_PARTS[..].iter()
+    V3_PARTS[..].iter()
```

Comment:
I think changing all these variables to the latest is annoying. Should just have a struct for each schema that contains parts, idents, etc., and just reference latest via `const LATEST = V3_Schema` and use that throughout.